.github/ISSUE_TEMPLATE/gc_member_onboarding.md

```diff
@@ -0,0 +1,200 @@
+Group have write access to the OpenTelemetry public calendar. To view and manage
+members of this group one must use the `opentelemetry.calendar.owner@gmail.com`
+account.
+
+The existing member must:
+
+- [ ] Add user as `Owner` of https://groups.google.com/g/opentelemetry-calendar-contributors.
+
+The new member must:
+
+- [ ] Validate they have `Owner` permissions on the group above.
+
+### Zapier
+Zapier is used to sync meeting recordings from Zoom to a publicly viewable
+Google spreadsheet (see https://github.com/open-telemetry/community/blob/main/docs/how-meeting-recordings-upload-works.md
+for more details).
+
+The new member must:
+- [ ] Validate they can log in to https://zapier.com with the credentials in 1Password.
+
+## Meetings and Ceremonies
+As per GC charter, all members are expected to attend the following meetings.
+
+- Governance Committee Meeting (every Thursday 11:00 Pacific Time, which is
+  typically 19:00 UTC).
+- Governance/Technical Committee Joint Meeting (every second Thursday of the month, coinciding 
+  with the Governance Committee Meeting).
+
+The existing member must:
+- [ ] Invite the new member to the meetings using the new members' preferred
+  email address.
+
+## Further Reading
+The following is a list of documents and other media that new members must
+familiarize themselves with. The new member must go through these media and ask
+any relevant questions that will aid them in understanding their role as a GC
+member. The existing member must assist the new member, to the best of their
+capacity, in finding answers to these questions.
+
+- [ ] [OpenTelemetry Code of Conduct](https://github.com/open-telemetry/community/blob/main/code-of-conduct.md):
+  all GC members define, evolve, and defend the Code of Conduct.
+- [ ] [OpenTelemetry Governance Committee meeting notes](https://docs.google.com/document/d/1-23Sf7-xZK3OL5Ogv2pK0NP9YotlSa0PKU9bvvtQwp8):
+  to identify items currently in progress. [Meeting Recordings](https://docs.google.com/spreadsheets/d/1SYKfjYhZdm2Wh2Cl6KVQalKg_m4NhTPZqq-8SzEVO6s)
+  can help get more details if needed.
+- [ ] [Project Management](https://github.com/open-telemetry/community/blob/main/project-management.md):
+  to understand the fundamentals of how projects are managed by the GC and TC.
+- [ ] [Active Projects](https://github.com/open-telemetry/community/tree/main/projects):
+  to understand current project deliverables and the challenges they aim to
+  solve.
+- [ ] [Project Board](https://github.com/orgs/open-telemetry/projects/29):
+  including the individual project boards for each of these, they help
+  understand the current state of the projects listed in the previous item.
+- [ ] [Community repo docs](https://github.com/open-telemetry/community/tree/main/docs):
+  for instructions on how to work with calendars, configuring repositories,
+  uploading recordings, etc. As a GC member, it is useful to know that these 
+  docs are there when needed.
```

Comment:
```suggestion
  docs are there when needed.
- [ ] [Inclusive Open Source Community Orientation (LFC102)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/): This free orientation course is designed to provide you with essential background knowledge and practical skills to create an inclusive culture in the open source community. Members of the GC may take this course in support of our community values.
```

@danielgblanco here's the training from LF I was talking about, we might discuss at a later stage how mandatory this should be for leadership roles in the otel community, but for now we can have it as a recommendation, wdyt?

(I enrolled in this course last week and did the first ~20 min so far, will share my thoughts with the GC after I went through all the material)

For reference, K8s steering has the following statement:  https://github.com/kubernetes/steering/blob/main/charter.md#inclusive-leadership-training

Reply:
Yeah, I like that idea